Documentation/UserRoundTrip.rst

```diff
@@ -0,0 +1,345 @@
+   | **Read online:** | https://docs.typo3.org/p/mask/mask/main/en-us/ |
+   | **TER:**         | https://extensions.typo3.org/extension/mask    |
+
+and find the README file usually displayed by the VCS host below the file list
+of the repository (see `Mask example <https://github.com/Gernott/mask>`__).
+
+For a Composer package, replace the TER URL with the Packagist URL and label it
+"Packagist:". For a standalone manual, remove the TER entry.
+
+The Packagist URL is linked indirectly from the TER page and is therefore not
+included in this configuration file.
+
+
+.. _user-round-trip-packagist-aspect:
+
+Packagist
+---------
+
+Guide the user from the Packagist page to the other aspects by setting these
+:file:`composer.json` configuration values:
+
+.. code-block:: none
+
+    "homepage": "<typo3-extension-repository>",
+    "support": {
+        "docs": "<documentation>",
+        "issues": "<vcs-repository-issues>",
+        "source": "<vcs-repository>"
+    }
+
+for example for the *Address List* extension:
+
+.. code-block:: none
+
+   "homepage": "https://extensions.typo3.org/extension/tt_address",
+   "support": {
+      "docs": "https://docs.typo3.org/p/friendsoftypo3/tt-address/main/en-us/",
+      "issues": "https://github.com/FriendsOfTYPO3/tt_address/issues",
+      "source": "https://github.com/FriendsOfTYPO3/tt_address"
+   }
+
+and find the links to the other aspects shown in the right column of the
+Packagist page (see `Address List example <https://packagist.org/packages/friendsoftypo3/tt-address>`__).
+
+For a Composer package, replace the TER URL with the Packagist URL.
+
+The issues URL is considered here because it is closely related to the VCS
+repository aspect.
+
+
+.. _user-round-trip-ter-aspect:
+
+TYPO3 Extension Repository
+--------------------------
+
+Guide the user from the TER page to the other aspects by setting these
+configuration values on the
+`extension management page <https://extensions.typo3.org/my-extensions>`__:
+
+*  **Published on Packagist:** check
+*  **Composer name of extension:** `<package-name>`
+*  **Link to issue tracker:** `<vcs-repository-issues>`
+*  **Link to repository:** `<vcs-repository>`
+
+for example for the *Static File Cache* extension:
+
+*  **Published on Packagist:** check
+*  **Composer name of extension:** lochmueller/staticfilecache
+*  **Link to issue tracker:** \https://github.com/lochmueller/staticfilecache/issues
+*  **Link to repository:** \https://github.com/lochmueller/staticfilecache
+
+and find the links to the other aspects presented in the right column of the TER
+page (see `Static File Cache example <https://extensions.typo3.org/extension/staticfilecache>`__).
+
+The documentation URL is automatically resolved when the documentation is
+published on docs.typo3.org. The issues URL is considered here because it is
+closely related to the VCS repository aspect.
+
+
+.. _user-round-trip-project-types:
+
+Project types
+=============
+
```

Comment:
Something seems to be missing here...?

Reply:
Added an introduction sentence to both sections "Project types" and "Project aspects".